chapters/annotations.tex

```diff
@@ -187,8 +189,11 @@ \subsubsection{Axis Properties}\doublelabel{axis-properties}
 When an axis bound isn't provided, the tool computes one automatically.
 
 The Modelica tool is responsible for showing the unit used for values at the
-axis tick marks, so the axis \lstinline!label! shall not be used to convey this
-information.
+axis tick marks, so the axis \lstinline!label! shall not contain the unit.
+
+If the specified unit is not supported by the tool, or the unit does not
+represent a display unit of the plotted variable (for example,
+\lstinline!unit="cm"! and variable denotes a mass), the behavior is unspecified.
```

Comment:
Again, the use of _unspecified_ opens up for various tools giving this a nice treatment, which will result in figures developed with different tools to become non-portable without a clear statement in the specification saying that the figures are incorrect.

Reply:
Looks good now.